src/common/components/clusterWizard/networkingSteps/ManagedNetworkingControlGroup.tsx

```diff
@@ -7,19 +7,28 @@ import { useTranslation } from '../../../hooks/use-translation-wrapper';
 export interface ManagedNetworkingControlGroupProps {
   disabled: boolean;
   tooltip?: string;
+  isUmnDisabled?: boolean;
 }
 
 const GROUP_NAME = 'managedNetworkingType';
 export const ManagedNetworkingControlGroup = ({
   disabled = false,
   tooltip,
+  isUmnDisabled,
 }: ManagedNetworkingControlGroupProps) => {
   const tooltipProps: TooltipProps = {
     hidden: !tooltip || !disabled,
     content: tooltip,
     position: 'top',
   };
+  const tooltipPropsUmnDisabled: TooltipProps = {
```

Comment:
```suggestion
  const tooltipPropsUmnDisabled: TooltipProps = {
    hidden: true,
    content: t('ai:User-Managed Networking is not supported when using Nutanix'),
    position: 'top',
  };
```



The variable `tooltipUmnDisabled` is used before it's declared. I think we can't just add the text when we declare the object.

Reply:
True. Is a mistake.